### DISCUSSION TITLE
¿Te interesaría asistir a un horario de oficina mensual para hablar sobre Angular en castellano?

### DISCUSSION THREAD

**Discussion by Splaktar** [angular-hispano/angular-hispano#161] [General]
Sería una videollamada a través de Google Meet. El formato sería bastante abierto. Podríamos hablar sobre las últimas actualizaciones en lanzamientos de Angular, hablar sobre casos de uso y desafíos para implementar características, o simplemente hablar sobre eventos actuales en la comunidad y / o el mundo.

¿Estarías interesado?

¿Qué otros temas te gustaría discutir?

**Comment by AXEL1988** [comment]
Muy interesado!
Sería interesante hablar más sobre buenas prácticas y arquitectura de aplicaciones en proyectos de gran escala basados en Angular.

**Comment by Villanuevand** [comment]
Yo creo que sería de una gran ayuda, se podría hablar de cualquier tema o simplemente entrar y proponer algún tema puntual.

Lo idea sería tener siempre un punto a discutir . A mí me gustaría aprender a como crear una librería con la arquitectura de material y como meterse en código de un proyecto open source para colaborar.

**Comment by LuisReinoso** [comment]
Sería super útil aumentar los skills de angular de todos. Como dice @Villanuevand siempre con un tema puntual no importa que sea algo o muy básico o avanzado. Con un tema puntual sería excelente.

**Comment by Villanuevand** [comment]
Creo que podrían haber 2 opciones de Hora de Oficina, 

- Tema Puntual
- Conversación Libre 

## Tema Puntual 

Debería hablarse sobre algún tema de interés general para la comunidad.

## Conversación Libre 

Hablar sobre algún tema que los miembros de las comunidad quiera hablar o comentar para desarrollar.

**Comment by Splaktar** [comment]
¿Qué días de la semana o mes funcionan mejor para usted?
¿Qué hora y zona horaria sería mejor para usted?

**Reply by Splaktar** [reply]
> ¿Qué días de la semana o mes funcionan mejor para usted?
> ¿Qué hora y zona horaria sería mejor para usted?

Algien?

**Reply by Villanuevand** [reply]
Por aqui... hay gente! xD

**Comment by Villanuevand** [comment]
Creo que deberíamos empezar a usar el horario de oficina para afinar el proceso de documentación? 

Propongo 1 vez a la semana pudiendo ser los siguientes horarios: 

### Martes

🇪🇸: 18:00 - (GMT+2)
🇦🇷🇧🇷🇺🇾:  13:00 - (GMT-3)
🇧🇴🇨🇱🇵🇾🇩🇴🇻🇪: 12:00  - (GMT-4)
🇨🇴🇪🇨🇲🇽🇵🇦🇵🇪: 11:00 - (GMT-5)
🇨🇷🇭🇳🇬🇹🇸🇻: 10:00 - (GMT-6)

### Miercoles 
🇪🇸: 16:00 - (GMT+2)
🇦🇷🇧🇷🇺🇾:  11:00 - (GMT-3)
🇧🇴🇨🇱🇵🇾🇩🇴🇻🇪: 10:00  - (GMT-4)
🇨🇴🇪🇨🇲🇽🇵🇦🇵🇪: 09:00 - (GMT-5)
🇨🇷🇭🇳🇬🇹🇸🇻: 08:00 - (GMT-6)

**Comment by Splaktar** [comment]
Gracias por proponerme algunas horarios y días. No puedo asistir los martes a 12:00 EDT (GMT-4). Tampoco 10:00 EDT en los Miercoles.

Para mí, estoy más disponible por las tardes, pero supongo que eso no funcionará para las personas en España (GMT+2).

Es posible que no pueda atender llamadas semanales, todas las semanas, pero estoy feliz de darle una oportunidad para ver cómo va.

Según mi calendario actual, podría hacer los siguientes horarios (GMT-4): 
- miércoles a las 12:00 o 14:00
- jueves a las 14:00 
- viernes a las 14:00

**Comment by devpato** [comment]
@Splaktar @Villanuevand  las juntas de docs-core en la junta del miercoles quedamos que iban a ser cada 15 dias a la misma hora al menos que varias personas no pudieran. Tambien creo que si tenemos planeado hacer juntas para los contribuyentes creo que una vez a la semana es demasiado

**Reply by Splaktar** [reply]
Como referencia, dado que muchas personas no pudieron asistir, eso fue a las 12:00 EDT (GMT-4).

**Reply by Splaktar** [reply]
¿La próxima reunión de docs-core sería el 5 de agosto a las 12:00 EDT (GMT-4) entonces? Es decir. cada 2 semanas. ¿O sería el 6 de agosto para cumplir los 15 días?

**Reply by devpato** [reply]
Miercoles no? para hacerlo mas consistente...o que piensas?

**Reply by Villanuevand** [reply]
El horario me parece bastante bueno, yo podría asistir sin problemas. 

Deberíamos hacer quizás un mail list para agendar en calendar el evento, me paso que falté a la anterior por que me aviso un día antes y muchas horas antes del evento creo que unas 4. 

Podríamos poner 1 día antes y unos 15 min antes.

**Reply by devpato** [reply]
hahahaha ese fue mi error mi estimado Andres! queria que estuvieras ready y peinado 1 dia antes haha :)

**Reply by Villanuevand** [reply]
@devpato  El dia de mi boda tenía alarma cada 2 horas empezando 8 horas antes del evento... xD
